src/i18n.ts

```diff
@@ -12,8 +11,10 @@ const buildI18n = () => {
     fallbackLng: 'en',
     // debug only when not in production
     debug: process.env.NODE_ENV !== 'production',
+    // define accessible namespaces
     ns: ['translations'],
```

Comment:
You don't need to define `chatbox` here?

Reply:
@pyphilia I think this would add the namespace to the list of loaded namespaces... not sure, maybe we could include all the name space in there but that would mean we do not get a gain of performance ... What do you think ? Maybe we should have a second parameter to the function telling which namespaces to include ? and if none were provided we just include the default one.